enos/modules/k8s_vault_verify_version/scripts/get-status.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+set -e
+
+status=$(${VAULT_BIN_PATH} status -format=json)
+version=$(${VAULT_BIN_PATH} version)
+
+echo "{\"status\": ${status}, \"version\": \"${version}\"}"
```

Comment:
```suggestion
echo "{\"status\": ${status}, \"version\": \"${version}\"}"

```

Reply:
Feel free to ignore- I just saw that it had the "no newline at end of file" sign on it.

Reply:
ah, that's the difference. Yeah I'll add the newline